permissions-policy-client-hints.md

```diff
@@ -0,0 +1,273 @@
+
+Permissions Policy cannot be used to change the hints received by the top-level
+document, as the `Permissions-Policy` response header cannot be sent before the
+hints (which are request headers) are received. However, it can be used to
+control where those hints are sent after that.
+
+### Delegation to embedded documents
+
+Any document which receives hints may choose to delegate those hints to other
+documents which it embeds inside of an `<iframe>` element.
+
+By default, low-entropy hints are sent to all embedded documents, while
+high-entropy hints are only sent if the embedded document is same-origin with
+its embedder ([Example 1](#example-1)). The iframe's `allow` attribute is used to
+name specific hints which should be sent to the embedded document
+([Example 2](#example-2)).
+
+> **Question:** Does the embeddded document's `Accept-CH` header affect this
+> processing at all?
+
+If an embedded document is same-origin with its embedder, then Permissions
+Policy will delegate all available hints by default. In that case, the `allow`
+attribute can still be useful, both for restricting hints which would otherwise
+be sent ([Example 3](#example-3)), and also for controlling hints when the iframe
+is navigated away from its initial origin ([Example 4](#example-4)).
+
+The `Permissions-Policy` header is also considered when delegating hints to
+embedded documents. If a hint is named in a document's `Permissions-Policy`
+header, then the corresponding allowlist is also checked, and the embedded
+document's origin must *also* be included in that list in order for it to be
+sent hints ([Example 5](#example-5)).
+
+Note that any hints which were not sent to a document *cannot* be sent to any
+documents which it embeds.
+
+### Delegation to subresources
+
+Client hints are also sent for subresource requests from a document, for
+resources such as images, stylesheets and scripts. The elements used to embed
+these resources (`<img>`, `<link>` and `<script>` elements) do not support an
+`allow` attribute, and so the processing is slightly different in this case.
+
+For hints to be sent to an origin server for a subresource request, the
+`Permissions-Policy` header is checked in every case. By default, any
+low-entropy hints with the embedding document received will be sent with all
+requests, while high-entropy hints are only sent if the resource is same-origin
+with the document ([Example 6](#example-6)).
+
+By using the `Permissions-Policy` header, it is possible to name the origins for
+which subresrouces should (or by omission, should not) receive hints
+([Example 7](#example-7)).
+
+Note that any hints which were not sent to the document in the first place
+*cannot* be sent with any subresource requests, regardless of origin.
+
+## Examples
+
+### Example 1: Default behaviour of delegation to frames
+
+Resource: https://example.com/
+
+Response headers:
+```http
+Accept-CH: Sec-CH-DPR
```

Comment:
`Accept-CH` doesn't include the `Sec-` and `CH-` prefixes.

Reply:
Prefixes (prefices?) removed, here and elsewhere.